status-im/status-react
# Add support for $UBI token

fixes #12975 

### Summary

Adding Universal Basic Income token ($UBI) to our default token list

<img width="448" src="https://user-images.githubusercontent.com/18485527/148233757-4c323193-ed86-4ab1-a3c5-ff82f82ba54e.png">

<img width="448" src="https://user-images.githubusercontent.com/18485527/148233767-2aeaa37a-04a8-4125-abac-1fb82cd18978.png">

#### Platforms

- Android
- iOS

##### Functional

- wallet / transactions

### Steps to test

- Open Status
- Go to wallet
- Open option
- Select Manage assets
- Verify that Universal Basic Income (UBI) is available to select and use

status: ready
